### PR TITLE
Request Portfolio Peer Review by Mike

### PR DESCRIPTION
**Pull request by davin2020**
Pls review things I have done -

- some media queries for wider & narrower screens
- tested in browsers Ch FF Sf Edge wrt overall width & aesthetics
- validated html
- included content about me & my work/experience & own photos - hence no img copyright issues

Things I haven't done or could improve -
- replaced `a` tags with `id` for internal anchors
- more detailed content for About and Portfolio sections
-make images within Portfolio section the same overall size & aspect ratio

FYI - used HSL as my color system so I can easily adjust related colors in terms of light/saturation properties

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+					<article class="articleBox articleBorder">
+						<h2>Service Delivery</h2>
+						<p class="justifyLeft">Previous service desk application experience</p>
+							<ul class="justifyLeft">
+								<li>ServiceNow, Ivanti/HEAT, VMWare, LANDesk, JIRA</li>
+								<li>Windows 10, 7, XP and Server 2008</li>
+								<li>Mac OSX</li>
+								<li>Active Directory, Exchange (hybrid OnPrem & cloud), Azure AD licensing, SharePoint, Skype</li>
+								<li>Kaseya VSA - ITSM software used to manage other computers</li>
+
```
Comment (mporam) [inline index.html:125]: unnecessary line break

File: index.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+								<li>Supporting corporate endusers in a windows environment, both onsite & field services</li>
+							</ul>
+					</article>
+				</div>
+			</section>
+			
+<!-- Section: Contact-->
+			<section class="colorBackgroundLight bkgImageStyle bkgImagePicContact">
+				<div class="opacityBkgImages">
+				<a name="Contact">
```
Comment (mporam) [inline index.html:135]: incorrect indentation here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+				<h1 class="fontCursive colorFontSections">Contact</h1>
+				</a>
+				<div class="containerFlex">
+					<article class="articleBox">
+						<a href = "mailto:davin.hayden@outlook.com"><i class="fas fa-envelope-square fa-3x"></i><br>davin.hayden@outlook.com</a>
```
Comment (mporam) [inline index.html:140]: remove spaces either side of `=`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  <link href="style.css" type="text/css" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
+</head>
+
+<body>
+	<div class="mainContainerGrid">
+<!-- Header & Navigation -->
+		<header class="colorHeaderFooter fontCursive">
+			<nav class="fontSizeNav ">
```
Comment (mporam) [inline index.html:17]: unnecessary space after class before `"`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+  <link href="normalize.css" type="text/css" rel="stylesheet">
+  <link href="style.css" type="text/css" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
+</head>
+
+<body>
+	<div class="mainContainerGrid">
+<!-- Header & Navigation -->
+		<header class="colorHeaderFooter fontCursive">
```
Comment (mporam) [inline index.html:16]: these classes are not semantic, you should decrobe what the thing IS not what it looks like. its not a header and a footer, its certainly not a color

Comment (mporam) [reply index.html:16]: most of your classes are not semantic throughout

File: style.css [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	background-color:  hsl(204, 46%, 20%);
+	font-family: Verdana;
+	font-size: 90%;
+
```
Comment (mporam) [inline style.css:18]: unnecessary line break

File: style.css [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+---------------------------------------------------------------- */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	background-color:  hsl(204, 46%, 20%);
```
Comment (mporam) [inline style.css:15]: please use hex codes for colors

File: style.css [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+	
+}
+
+
+/* ---------------------------------------------------------------------------
+	Generic Classes for  Containers & Sections and Cursive Font for Sections
+---------------------------------------------------------------------------- */
+
+.mainContainerGrid {
+	/* nothing required here, only MQ later */
```
Comment (mporam) [inline style.css:27]: if nothing required, please remove the selector

File: style.css [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+@media screen and (min-width: 1080px) {
+	.mainContainerGrid {
+		width: 80%;
+		margin: 0 auto;
+	}
+}
+
+.colorBackgroundDark {
+	/* darker stone blue - using HSL so can easily relate color-lightness */
+	background-color:  hsl(204, 46%, 44%);
```
Comment (mporam) [inline style.css:39]: again, hex codes for colors everywhere, except when doing opacity, where rgba is okay

File: style.css [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+	HEADER & NAV 
+--------------------------------------------- */
+
+header {
+	text-align: center;
+	font-size: 120%;
+}
+
+.colorHeaderFooter {
+	background-color: rgb(87, 24, 103);
```
Comment (mporam) [inline style.css:70]: hex

File: style.css [hunk 11]
```diff
@@ -0,0 +1,278 @@
...
+	color: white;
```
Comment (mporam) [inline style.css:71]: hex

File: style.css [hunk 12]
```diff
@@ -0,0 +1,278 @@
...
+/* ---------------------------------------------------------------------
+	MAIN Section with background image & containers & content formatting 
+------------------------------------------------------------------------ */
+
+.bkgImageStyle {
+	width: 100%;
+	height: auto;	
+	background-size: cover; /* user cover to get full width, instead of contain */
+	background-repeat: no-repeat;
+
```
Comment (mporam) [inline style.css:122]: remove unnecessary line breaks

File: style.css [hunk 13]
```diff
@@ -0,0 +1,278 @@
...
+	background-repeat: no-repeat;
+	
+}
+
+.opacityBkgImages {
+	background-color: rgba(61,123,164,0.7);
+	padding-top: 10px;
+	padding-bottom: 20px;
+	
+}
```
Comment (mporam) [inline style.css:130]: missing line break after closing `}`

File: style.css [hunk 14]
```diff
@@ -0,0 +1,278 @@
...
+
+@media screen and (max-width: 550px) {
+	.contentIntro {
+		padding-left: 150px;
+	}
+}
+
+.textAboutMe {
+	color: white;
+/*	background-color: black;*/
```
Comment (mporam) [inline style.css:187]: remove if not needed

File: style.css [hunk 15]
```diff
@@ -0,0 +1,278 @@
...
+	background-color: rgb(87, 24, 103);
+	vertical-align: middle;
+/* need to set top & bottom margins to 0 so there is no space between the text for the Name and Role */
+	margin-top: 0px;
```
Comment (mporam) [inline style.css:191]: 0 values dont need px, 0 is 0 in any unit

**Review by mporam — CHANGES_REQUESTED** [review]
also suggest looking at the hover state of your navigation items

File: index.html (hunk 1 above)
Comment (davin2020) [reply index.html:125]: removed

File: index.html (hunk 2 above)
Comment (davin2020) [reply index.html:135]: indented more

File: index.html (hunk 3 above)
Comment (davin2020) [reply index.html:140]: done

File: index.html (hunk 4 above)
Comment (davin2020) [reply index.html:17]: removed

File: style.css (hunk 6 above)
Comment (davin2020) [reply style.css:18]: removed

File: style.css (hunk 8 above)
Comment (davin2020) [reply style.css:27]: removed

File: style.css (hunk 12 above)
Comment (davin2020) [reply style.css:122]: removed

File: style.css (hunk 13 above)
Comment (davin2020) [reply style.css:130]: added

File: style.css (hunk 14 above)
Comment (davin2020) [reply style.css:187]: removed

File: style.css (hunk 15 above)
Comment (davin2020) [reply style.css:191]: done

File: index.html (hunk 1 above)
Comment (davin2020) [reply index.html:125]: made nav hover better

File: style.css (hunk 7 above)
Comment (davin2020) [reply style.css:15]: replaced hsl & rgb with hex

**Review by mporam — APPROVED** [review]

**Pull request merged**